ddtrace/span.py

```diff
@@ -474,7 +474,7 @@ def set_exc_info(self, exc_type, exc_val, exc_tb):
         # readable version of type (e.g. exceptions.ZeroDivisionError)
         exc_type_str = "%s.%s" % (exc_type.__module__, exc_type.__name__)
 
-        self.meta[ERROR_MSG] = ensure_text(exc_val.message)
+        self.meta[ERROR_MSG] = ensure_text(exc_val.message) if six.PY2 else stringify(exc_value)
```

Comment:
@P403n1x87 should this be covered by `ensure_text`? that is `if six.PY2 ... ` part.

Reply:
@brettlangdon I believe `stringify(exc_val)` does the sensible thing on PY3, so this should be good
@juan-fernandez sorry I had a typo in my suggestion! 😢 
```suggestion
        self.meta[ERROR_MSG] = ensure_text(exc_val.message) if six.PY2 else stringify(exc_val)
```